projects/plugins/jetpack/modules/publicize/publicize.php

```diff
@@ -641,14 +757,13 @@ public function get_filtered_connection_data( $selected_post_id = null ) {
 
 				$unique_id = $this->get_connection_unique_id( $connection );
 
-
 				// Was this connection (OR, old-format service) already Publicized to?
 				$done = ! empty( $post ) && (
-					// New flags
-					1 == get_post_meta( $post->ID, $this->POST_DONE . $unique_id, true )
+					// New flags.
+					1 === (int) get_post_meta( $post->ID, $this->POST_DONE . $unique_id, true )
```

Comment:
How come the check isn't failing on POST_DONE here?

Reply:
That's actually because of [the rule being ignored by the entire file](https://github.com/Automattic/jetpack/blob/master/projects/plugins/jetpack/modules/publicize/publicize.php#L2). Perhaps I should take the same approach for the other file, and we can change the name of all constants down the road.

Reply:
Decided to keep the stricter checks in place rather than ignoring the rule for the entire file